extension/package.json

```diff
@@ -552,6 +552,12 @@
                     "default": true,
                     "description": "Specifies if symbols should be loaded from the .alpackages folder. This is used when documentation is generated, ToolTips are added etc.",
                     "scope": "resource"
+                },
+                "NAB.RefreshXlfAfterFindNextUntranslated": {
+                    "type": "boolean",
+                    "default":false,
```

Comment:
no auto-format?
```suggestion
                    "default": false,
```

Reply:
format on save was set to `false` in workspace, do you remember if we had a reason for that? I've changed it now, so I guess we'll get a quick reminder :) I also added a default formatter, see workspace file.